src/api/Web.Bff.Shopping/Config/CommentApiOperations.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Web.Bff.Shopping.Config
+{
+    public partial class UrlsConfig
+    {
+        public class CommentApiOperations
+        {
+            public static string AddComment() => $"/api/comments/";
```

Comment:
This string has no reason to be a method type.
- [ ] replace with ``` public static string AddComment = $"/api/comments/";```

Reply:
Also, did you test it? When I was making the similar task with Products, I've removed these starting slashes, as I was always adding the URL to the appsettings.json with the slash in the end, so requests had the wrong URL when baseUrl and resource are concatenated.